projects/ssr-tests/src/ssr-testing.spec.ts

```diff
@@ -33,15 +33,18 @@ describe('SSR E2E', () => {
     ]);
   });
 
-  // TODO: Test incomplete
-  xit('should receive cached response with next request', async () => {
+  it('should receive cached response with next request', async () => {
```

Comment:
```suggestion
  it('should receive cached response with next request if previous request timed out', async () => {
```

Reply:
I suppose it depends on [this question](https://github.com/SAP/spartacus/pull/17967/files#r1365017454). I was thinking to rename it to `should receive cached response with next request if ssr engine completes rendering`.